tools/db_bench_tool.cc

```diff
@@ -351,6 +352,46 @@ DEFINE_uint32(overwrite_window_size, 1,
               "Warning: large values can affect throughput. "
               "Valid overwrite_window_size values: [1, kMaxUint32].");
 
+DEFINE_uint64(
+    disposable_entries_delete_delay, 0,
```

Comment:
I am slightly concerned about "polluting" the global option space with benchmark-specific parameters, but I don't have a good suggested alternative. But please clearly mark these as only (currently) applying to filluniquerandom. Or would it make sense to make this a new benchmark like fillanddeleteuniquerandom? Not sure, legit asking

Reply:
This is also something I've been struggling with. 
I can totally create a new benchmark "fillanddeleteuniquerandom" [but this is very superficial since we still have to use the DoWrite function under the hood (used by all the fill... benchmarks)], but in all the scenarios I've thought of, we still end up having these 5 new option flags hanging around...
For now I can create this new benchmark and clearly mark that these flags only apply to this new benchmark (that will make explanations clearer), and I will explicitly mention it for each flag.